native/SampleApps/RestAPIExplorer/RestAPIExplorer/AppDelegate.swift

```diff
@@ -0,0 +1,259 @@
+/*
+ AppDelegate.swift
+ RestAPIExplorerSwift
+ 
+ Created by Nicholas McDonald on 1/3/18.
+ 
+ Copyright (c) 2018-present, salesforce.com, inc. All rights reserved.
+ 
+ Redistribution and use of this software in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this list of conditions
+ and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice, this list of
+ conditions and the following disclaimer in the documentation and/or other materials provided
+ with the distribution.
+ * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+ endorse or promote products derived from this software without specific prior written
+ permission of salesforce.com, inc.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import UIKit
+import SalesforceSDKCore
+import SalesforceSwiftSDK
+import MobileCoreServices
+
+// Fill these in when creating a new Connected Application on Force.com
+let RemoteAccessConsumerKey = "3MVG9g9rbsTkKnAWkkUnzOrokwEMnYtP.O0E2c_5E7Xe9QAaTHg3R.pzB7Y_itz_KnTzUSym9hKA9dA2yfg7X"
+let OAuthRedirectURI        = "com.salesforce.mobilesdk.sample.restapiexplorer://oauth2/success"
+
+
+@UIApplicationMain
+class AppDelegate : UIResponder, UIApplicationDelegate
+{
+    var window: UIWindow?
+    
+    override
+    init()
+    {
+        super.init()
+        
+        /*
+         // scheme of idpApp
```

Comment:
Remove commented out code.

Reply:
@bhariharan Why remove it? Its also present [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/9c51f4c89424878ed1acb6405f9d5e2e8784f5fa/native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m#L50)

Reply:
My bad. Maybe add comments on when it needs to be commented out then?